terraform/environment/shared_data_sources.tf

```diff
@@ -0,0 +1,7 @@
+data "aws_security_group" "redis_cache_service" {
+  name = "redis-cache-service"
+}
+
+data "aws_elasticache_replication_group" "redis_cache" {
+  replication_group_id = "redis-cache-replication-group"
+}
```

Comment:
add a railing newline. might be worth running `terraform fmt` over your environment folder also...